libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -297,12 +302,57 @@ private void doLoadPage(boolean jwtFlow) {
         try {
             URI uri = getAuthorizationUrl(jwtFlow);
             callback.loadingLoginPage(loginOptions.getLoginUrl());
-            webview.loadUrl(uri.toString());
+            if (SalesforceSDKManager.getInstance().isBrowserLoginEnabled()) {
+                loadLoginPageInChrome(uri);
+            } else {
+                webview.loadUrl(uri.toString());
+            }
         } catch (URISyntaxException ex) {
             showError(ex);
         }
     }
 
+    private void loadLoginPageInChrome(URI uri) {
+        final Uri url = Uri.parse(uri.toString());
+        final CustomTabsIntent.Builder intentBuilder = new CustomTabsIntent.Builder();
+
+        /*
+         * Sets custom animation to slide in and out for Chrome custom tab so that
+         * it doesn't look like a swizzle out of the app and back in.
+         */
+        intentBuilder.setStartAnimations(activity, android.R.anim.slide_in_left,
```

Comment:
Custom animations to minimize swizzle effect and make it look more like another screen within the app.